Makefile

```diff
@@ -169,7 +169,8 @@ bundle: generate manifests download-kustomize download-operator-sdk ## Generate
 
 	CSV_PATH=$$($(MAKE) csv-path)
 	yq -riY '.metadata.annotations.containerImage = "'$(IMG)'"' $${CSV_PATH}
-	yq -riY '.spec.install.spec.deployments[0].spec.template.spec.containers[1].image = "'$(IMG)'"' $${CSV_PATH}
+	# Update container image for container 'kuebrnetes-image-puller-operator' in the list of deployments
+	yq -riY '.spec.install.spec.deployments[0].spec.template.spec.containers[] |= (select(.name == "kubernetes-image-puller-operator") .image |= "'$(IMG)'")' $${CSV_PATH}
```

Comment:
When running `make bundle` on the dogfooding cluster, the order of the containers are different. This change makes it so that we don't assume that the `kubernetes-image-puller-operator` container is defined at index `1`